### ISSUE TITLE
Scrolling thumbnail pager?

### ISSUE THREAD

**Issue by andrews05** [stevenwanderski/bxslider-4#832]
Hi
I'm just looking at this plugin, haven't tried it yet but I'm wondering: does the pager support scrolling when there are many thumbnails? Like a mini-carousel of sorts.

**Comment by Tidal-Wave** [comment]
While not directly it is possible to make something like that with this library.

**Comment by andrews05** [comment]
Thanks, could you give me a rough idea of how to do that?
Is this a feature that could be added sometime?

**Comment by Tidal-Wave** [comment]
Using the callbacks you can chain two sliders together to create this effect, having one control the other. Thumbnails is definitely on my roadmap, but our next major feature enhancement is captions and data attributes.

**Comment by andrews05** [comment]
Great, thanks for that, I'll give it a go.

**Comment by andrews05** [comment]
Right, I've tried this out and for the most part it seems to "just work". The only problem I have is getting the thumbnail carousel to scroll automatically when the main slider changes.

The fundamental issue seems to be that for carousels the public functions 'goTo..', 'getCurrent..' etc work in pages, not individual slides. So if I have an index of a thumbnail in the carousel that I want to go to, I don't know how to do this because I don't know what page number the thumbnail is on.
Not sure what the best solution would be but perhaps I could work out the page number if the getMoveBy or getPagerQty methods were public?
